microsoft-edge/devtools-guide-chromium/index.md

```diff
@@ -139,13 +139,13 @@ In the Tools tab group, you may open the different tools that are available in t
 The DevTools provides lots of features and functionality to use with your web product.  Access the different parts of the DevTools in many ways, but the fastest way to access the features you need is to use the command menu.  For more information, navigate to [Run commands with the Microsoft Edge DevTools Command menu][DevtoolsGuideCommandMenuIndex].  To open the command menu, complete one of the following actions.  
 
 *   Select `Control`+`Shift`+`P` \(Windows, Linux\) or `Command`+`Shift`+`P` \(macOS\).  
-*   Choose **Customize And Control DevTools** \(`...`\), and then choose **Run Command**.  
+*   Select **Customize And Control DevTools** \(`...`\), and then select **Run Command**.  
 
 :::image type="complex" source="./media/devtools-intro-command-menu.msft.png" alt-text="Screenshot of the command menu in DevTools" lightbox="./media/devtools-intro-command-menu.msft.png":::  
 Screenshot of the command menu in DevTools  
 :::image-end:::  
 
-The command menu allows you to type commands to display, hide, or run features in the DevTools.  With the command menu open, enter the word **changes**, and then choose **Drawer Show Changes**.  The **Changes** tool opens which is useful when you edit CSS, but is difficult to find in the DevTools UI.  
+The command menu allows you to type commands to display, hide, or run features in the DevTools.  With the command menu open, enter the word **changes**, and then select **Show Changes**.  The **Changes** tool opens which is useful when you edit CSS, but is difficult to find in the DevTools UI.  
```

Comment:
```suggestion
The command menu allows you to type commands to display, hide, or run features in the DevTools.  With the command menu open, enter the word **changes**, and then select **Drawer: Show Changes**.  The **Changes** tool opens which is useful when you edit CSS, but is difficult to find in the DevTools UI.  
```
This is the convention I use.  It would be interesting to check repo convention, but regardless, I believe this (adding the command category eg "Drawer: " is most helpful to represent the UX.